_commands/basics/date.md

```diff
@@ -3,22 +3,58 @@
 
 date
 -------
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
-
+`date` returns a system time the user specified.
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ date
+Tue Feb  6 12:42:38 EST 2018
+$ date +%Y%m%d
+20180206
 ~~~
 
 <!--more-->
 
+This command returns the standard current time. With different options and parameters, it can also return other system time with different formats.
+
 ### Useful Options / Examples
 
+~~~ bash
+$ date --help
```

Comment:
No need to talk about the help flag since it is pretty common and exists for all commands